tests/phpunit/Application/QueryStringParserTest.php

```diff
@@ -177,15 +198,9 @@ public function testHandlesOrQuotedStrings(): void {
 		$this->assertSame( 'baz', $query->getFreeText() );
 	}
 
-	private function newQueryStringParser(): QueryStringParser {
+	private function newQueryStringParser( ?string $instanceOfId = 'P11111' ): QueryStringParser {
 		return new QueryStringParser(
-			instanceType: $this->newConfig()->getInstanceOfId()
-		);
-	}
-
-	private function newConfig(): Config {
-		return new Config(
-			instanceOfId: new NumericPropertyId( self::INSTANCE_TYPE_ID )
```

Comment:
Unsure why we built a Config just to get the ID

Reply:
I thought we have to replicate the constructor like the original class for tests. Is simplification without the Config acceptable?